### PR TITLE
Unit tests for evennia/utils/batchprocessors

### PR DESCRIPTION
**Pull request by Henddher**
Unit tests for `evennia/utils/batchprocessors`

Module coverage increased from 40.82% to 97.96%.

### CODE REVIEW

File: evennia/utils/batchprocessors.py [hunk 1]
```diff
@@ -233,7 +233,7 @@ def read_batchfile(pythonpath, file_ending=".py"):
                 continue
             break
     if not text and decoderr:
-        raise UnicodeDecodeError("\n".join(decoderr))
+        raise UnicodeDecodeError("\n".join(decoderr), bytearray(), 0, 0, '')
```
Comment (Henddher) [inline evennia/utils/batchprocessors.py]: UnicodeDecodeError constructor expects 5 args.

File: evennia/utils/batchprocessors.py [hunk 2]
```diff
@@ -145,7 +145,7 @@
 ```
 #HEADER
 
-from django.config import settings
+from django.conf import settings
```
Comment (Henddher) [inline evennia/utils/batchprocessors.py]: Documentation update.

**Comment by Henddher** [comment]
Hi @Griatch!

In the spirit of Hacktoberfest

File: evennia/utils/batchprocessors.py (hunk 1 above)
Comment (Griatch) [reply evennia/utils/batchprocessors.py]: Oh? Interesting. I was not aware of this change.

**Comment by Henddher** [comment]
Thanks @Griatch !

**Pull request merged**